igvm/vm.py

```diff
@@ -383,7 +383,7 @@ def shutdown(self, check_vm_up_on_transaction=True, transaction=None):
                 force_stop_failed=check_vm_up_on_transaction,
             )
 
-    def aws_shutdown(self, timeout: int = 120) -> None:
+    def aws_shutdown(self, timeout: int = 240) -> None:
```

Comment:
I'm not sure, if you want to rely on an _guessed_ increased timeout value in a DR case.
What about forcing a instance stop as a last resort instead of having a running instance left.
```
            response = self.ec2c.stop_instances(
                InstanceIds=[self.dataset_obj['aws_instance_id']],
                DryRun=False,
                Force=True
            )
```

Reference: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ec2.html#EC2.Client.stop_instances